src/client/package.json

```diff
@@ -57,12 +57,17 @@
   "devDependencies": {
     "autoprefixer": "6.3.1",
     "autoprefixer-loader": "3.1.0",
-    "babel": "5.8.23",
-    "babel-core": "5.8.33",
+    "babel": "6.5.2",
```

Comment:
it was.

On Wednesday, 20 April 2016, Ray Booysen <notifications@github.com> wrote:

> In src/client/package.json
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/277#discussion_r60452498>
> :
>
> > @@ -57,12 +57,17 @@
> >    "devDependencies": {
> >      "autoprefixer": "6.3.1",
> >      "autoprefixer-loader": "3.1.0",
> > -    "babel": "5.8.23",
> > -    "babel-core": "5.8.33",
> > +    "babel": "6.5.2",
>
> the shrinkwrap file needs to be updated.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/277/files/bc76237b96903c6cdb6b76700a25e83c55c34c5b#r60452498>
>


-- 
Dimitar Christoff

"JavaScript is to JAVA what hamster is to ham"
@D_mitar - https://github.com/DimitarChristoff